ietf-layer0-types.yang

```diff
@@ -1284,6 +1508,7 @@ module ietf-layer0-types {
       }
       case organizational-mode {
         container organizational-mode {
+          config false;
```

Comment:
If you change the config false in the "transmitter-tuning-range", be careful that that grouping is used also below in the case of explicit-mode. You should probably put a "config false" also under explicit-mode container not only under organizational-mode container . What do you think? If not in optical impairment you would have transmitter-tuning-range attributes configurable .

Reply:
Interesting finding - it looks like in optical impairment, the list "supported-mode" is already set to config-false, and this list is the only one that uses grouping "transceiver-mode", which in turn is the only one that uses "common-organizational-explicit-mode". So the config-false is carried over to the attributes in grouping "transmitter-tuning-range" regardless of what is set there. So there is no impact to the optical impairment model. Theoretically we could simply remove the config-false statement added.

Nevertheless I believe it'd be better to be safe to add config=false to both case statements. Will update it.